app/src/main/kotlin/com/simplemobiletools/calculator/helpers/BaseCalculatorImpl.kt

```diff
@@ -0,0 +1,89 @@
+package com.simplemobiletools.calculator.helpers
+
+import android.content.Context
+import com.simpletools.calculator.commons.R
+import com.simpletools.calculator.commons.helpers.Calculator
+import com.simpletools.calculator.commons.helpers.Formatter
+
+open class BaseCalculatorImpl(calculator: Calculator, open val context: Context) {
```

Comment:
what does open mean here

Reply:
It's used to declare an explicit supertype. This class is never instantiated. It's being inherited by two other helpers.

Reply:
I see